tests/typescript/declare/trailing-comma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`function-rest-trailing-comma.ts - {"trailingComma":"all"} format 1`] = `
+====================================options=====================================
+parsers: ["typescript"]
+printWidth: 80
+trailingComma: "all"
+                                                                                | printWidth
+=====================================input======================================
+declare function foo(...args: any[], )
+declare function foo(...long_long_long_long_long_long_long_long_long_long_long_long_long_long_long_args: any[], )
+
+=====================================output=====================================
+declare function foo(...args: any[]);
+declare function foo(
+  ...long_long_long_long_long_long_long_long_long_long_long_long_long_long_long_args: any[]
```

Comment:
We didn't print comma because https://github.com/prettier/prettier/blob/c42887265754931f98e16b6aadb49dab71df89ac/src/language-js/print/function-parameters.js#L152

Reply:
We shouldn't print it. The issue is only about parsing.